release-notes/v3.30.2-release-notes.md

```diff
@@ -0,0 +1,12 @@
+19 Jun 2025
+
+#### Bug fixes
+
+- ebpf: fix forwarding for asymetric routing https://github.com/projectcalico/calico/issues/10469 [calico 10535](https://github.com/projectcalico/calico/pull/10535) (@tomastigera)
+- Fix race condition in Goldmane startup [calico 10514](https://github.com/projectcalico/calico/pull/10514) (@caseydavenport)
+- Fix Goldmane race condition when terminating streams [calico 10513](https://github.com/projectcalico/calico/pull/10513) (@caseydavenport)
+
+#### Other changes
```

Comment:
we need to include the notes for these PRs as well:
- https://github.com/projectcalico/calico/pull/10518
- https://github.com/projectcalico/calico/pull/10532